packages/augur-ui/src/modules/app/actions/init-augur.ts

```diff
@@ -69,10 +69,18 @@ function pollForAccount(
     if (!attemptedLogin && connection.isConnected) {
       attemptedLogin = true;
 
-      const loggedInAccount = windowApp.localStorage.getItem('loggedInAccount');
-      const loggedInAccountType = windowApp.localStorage.getItem(
-        'loggedInAccountType'
-      );
+      let loggedInAccount = null;
+      let loggedInAccountType = null
+
+      try {
+        const loggedInUser =  JSON.parse(windowApp.localStorage.getItem('loggedInUser'));
+        loggedInAccount = loggedInUser && loggedInUser.address;
```

Comment:
is it "fine" if `loggedInAccount === false` ?

Seems better to test that we have a loggedInUser and then just extract the properties at that time.